api/src/test/java/uk/gov/dhsc/htbhf/claimant/ReportClaimIntegrationTest.java

```diff
@@ -55,25 +56,25 @@ void shouldReportNewClaimToGoogleAnalyticsWithPostcodeData() throws JsonProcessi
         String postcode = claim.getClaimant().getAddress().getPostcode();
         stubPostcodesIoAndGoogleAnalytics(postcode);
 
-        claimMessageSender.sendReportClaimMessage(claim, ONE_CHILD_UNDER_ONE_AND_ONE_CHILD_BETWEEN_ONE_AND_FOUR, ClaimAction.NEW);
+        claimMessageSender.sendReportClaimMessage(claim, ONE_CHILD_UNDER_ONE_AND_ONE_CHILD_BETWEEN_ONE_AND_FOUR, NEW);
         messageProcessorScheduler.processReportClaimMessages();
 
         wiremockManager.verifyPostcodesIoCalled(postcode);
-        wiremockManager.verifyGoogleAnalyticsCalledForClaimEvent(claim, ClaimAction.NEW, trackingId);
+        wiremockManager.verifyGoogleAnalyticsCalledForClaimEvent(claim, NEW, trackingId, ONE_CHILD_UNDER_ONE_AND_ONE_CHILD_BETWEEN_ONE_AND_FOUR);
     }
 
     @Test
     void shouldReportRejectedClaimToGoogleAnalyticsWithPostcodeData() throws JsonProcessingException {
-        Claim claim = aClaimWithClaimStatus(REJECTED);
+        Claim claim = aClaimWithClaimStatus(ClaimStatus.REJECTED);
         claimRepository.save(claim);
         String postcode = claim.getClaimant().getAddress().getPostcode();
         stubPostcodesIoAndGoogleAnalytics(postcode);
 
-        claimMessageSender.sendReportClaimMessage(claim, ONE_CHILD_UNDER_ONE_AND_ONE_CHILD_BETWEEN_ONE_AND_FOUR, ClaimAction.REJECTED);
+        claimMessageSender.sendReportClaimMessage(claim, ONE_CHILD_UNDER_ONE_AND_ONE_CHILD_BETWEEN_ONE_AND_FOUR, REJECTED);
         messageProcessorScheduler.processReportClaimMessages();
 
         wiremockManager.verifyPostcodesIoCalled(postcode);
-        wiremockManager.verifyGoogleAnalyticsCalledForClaimEvent(claim, ClaimAction.REJECTED, trackingId);
```

Comment:
Switched around the ClaimStatus and ClaimAction static import so that this line would be under 160 characters.